configs/_base_/datasets/pipeline/rand_aug.py

```diff
@@ -0,0 +1,42 @@
+rand_increasing_policies = [
```

Comment:
The  folder name should be ‘pipelines’.
Why using "increasing" in the name?

Reply:
`increasing` comes from `_RAND_INCREASING_TRANSFORMS` in timm. I have added a comment above that config.